README.md

```diff
@@ -6,42 +6,69 @@
 
 # ssi-service
 
-## Introduction
+HTTP-API driven web service that wraps our [ssi-sdk](https://github.com/TBD54566975/ssi-sdk).
```

Comment:
Add a bit more?
HTTP-API driven web service that wraps our ssi-sdk and allows interacting with the standards around Verifiable Credentials, Decentralized Identifiers and all things Self Sovereign Identity!

Reply:
The sentence below in Introduction section has some of that, let me know if you still want to add this.

Reply:
+1 on the the elevator pitch right at the top here. Suggestion pulling from both of y'all:

"A web service that exposes the `ssi-sdk` as an HTTP API. Support operations for Verifiable Credentials, Decentralized Identifiers and things Self Sovereign Identity!"

"HTTP-API driven web service" sounds semantically odd to me. :)

Reply:
And where andrew would you like me to add this, here or at the top?